### ISSUE TITLE
Multiple select - async callback with large dataset

### ISSUE THREAD

**Issue by lopsided** [OptimalBPM/angular-schema-form-dynamic-select#99]
This is a follow on issue to my last one: #86 

My list of labels has grown to be far too large to load them all up straight away so I set the `asyncCallback` to fetch just 20 items at a time from the server. This works with server-side filtering and works well. 

However when I save and go back to edit the item, my selections are kind of there but not populated with the details. I can see the plugin knows they are there but doesn't know what to show for them. 

I'm guessing this is because it can't find them in the 20 items fetched from the `asyncCallback`? 

Am I doing something wrong here or is this not supported?

**Comment by nicklasb** [comment]
Well, if the items aren't in the list, it can't match with them? Or is there something I don't understand?

**Comment by lopsided** [comment]
No, I think you understand, but I guess the plugin is not designed to cater for this. If that's the case then the only way to show the model selections is to load up all of the available options with the initial async request. This kind of defeats the point of using async load with server-side filtering and is not possible in my case with ~ 1million options. 

The select2 docs mention it in the faqs: https://select2.github.io/options.html#how-can-i-set-the-initially-selected-options-when-using-ajax. Though this is jquery based.

I will try to figure out a solution and hopefully get a PR for this.

**Comment by nicklasb** [comment]
Any progress?
